### ISSUE TITLE
Indefinite "Confirming match..." message resulting in bans

### ISSUE THREAD

**Issue by henke443** [ValveSoftware/Dota2-Gameplay#22326]
### Description

When searching for a game, there is a >10% chance that the matchmaking bugs out so that "Confirming match..." is displaying indefinitely, so that you are unable to accept the match. When restarting the client you will still be banned as if you failed to ready-up.

### Example Match ID (and possibly Timestamp)

-

### Screenshots

![image](https://github.com/user-attachments/assets/a0e15509-02a8-41ba-bc84-900889758938)

**Comment by henke443** [comment]
I suggest maybe temporarily disabling bans (or reducing the accumulative increases for the duration) for failing to ready-up until the real issue is fixed.

**Comment by Ali-Mahmoodi** [comment]
I have got same issue

**Comment by charli3cta** [comment]
I've got the same problem.

**Comment by Hosenbisler** [comment]
Same issue

**Comment by daudnadeem** [comment]
Same issue.

**Comment by ScythEwolf** [comment]
Similar issue here. I believe I have this since the last patch. So sad...

**Comment by simonmilz** [comment]
same issue here

**Comment by NoMore201** [comment]
I have this problem on EUW since PGL wallachia play offs, saturday 12 October, still to date...

**Comment by ScythEwolf** [comment]
Yes... it is on EUW - just found out that all you need to do is switching to another server.

**Comment by xbxb30** [comment]
Same problem... Stupid dota. 12 hour of ban!!!
